examples/advanced/xgboost_secure/README.md

```diff
@@ -43,16 +49,44 @@ For more details regarding federated XGBoost and the interface-plugin design, pl
 
 To run all experiments, we provide a script for all settings.
 ```
-bash run_training_local.sh
+bash run_training_standalone.sh
 ```
 This will cover baseline centralized training, local FL with and without secure feature.
 
-From the results, we can have three observations:
+## Run Federated Experiments with NVFlare
+Next, we run the federated XGBoost training without and with homomorphic encryption using NVFlare. 
+We run the NVFlare jobs using simulator with: 
+```
+bash run_training_fl.sh
+```
+The running time of each job depends mainly on the encryption workload. 
+
+## Results
+Comparing the AUC results with centralized baseline, we have four observations:
 1. The performance of the model trained with homomorphic encryption is identical to its counterpart without encryption.
-2. Vertical federated learnings have identical performance as the centralized baseline.
-3. Horizontal federated learnings have performance slightly different from the centralized baseline. This is because under horizontal FL, the local histogram quantiles are based on the local data distribution, which may not be the same as the global distribution.
+2. Vertical federated learning (both secure and non-secure) have identical performance as the centralized baseline.
+3. Horizontal federated learning (both secure and non-secure) have performance slightly different from the centralized baseline. This is because under horizontal FL, the local histogram quantiles are based on the local data distribution, which may not be the same as the global distribution.
+4. GPU leads to different results compared to CPU, which is expected as the GPU involves some data conversions.
```

Comment:
Do we need to explain in the script we just use default "cuda:0" for all clients?
In the documentatio we might need to show them how they can configure this.

@SYangster

Reply:
yes we can further update readme after adding the nvflare jobs